core/src/avm2/error.rs

```diff
@@ -38,12 +38,8 @@ impl Debug for Error<'_> {
 }
 
 // This type is used very frequently, so make sure it doesn't unexpectedly grow.
-#[cfg(target_family = "wasm")]
 const _: () = assert!(size_of::<Result<Value<'_>, Error<'_>>>() == 24);
```

Comment:
I thought that the size is smaller (16?) on non-wasm due to x86 being less strict with alignment?
At least at some point it was. Do we do any testing on 32-bit?

That's why I suggest keeping them separate here too.